test/specs/box.spec.js

```diff
@@ -1,14 +1,39 @@
 describe('Box annotation', function() {
   describe('auto', jasmine.fixtures('box'));
 
-  window.testEvents({
-    type: 'box',
-    id: 'test',
-    xMin: 2,
-    yMin: 2,
-    xMax: 4,
-    yMax: 4,
-    borderWidth: 0
-  });
+  describe('inRange', function() {
+    const annotation = {
+      type: 'box',
+      xMin: 2,
+      yMin: 4,
+      xMax: 8,
+      yMax: 6,
+      borderWidth: 0,
+      rotation: 0
+    };
+
+    const chart = window.scatter10x10({box: annotation});
```

Comment:
Is not better to pass only the annotation options and use in the scatter10x10 chart config the annotations as an array?

Reply:
I thought of leaving the choice of array vs object and how many annotations to the caller.